docs/concepts/controlling-catalog-selection.md

```diff
@@ -125,7 +125,7 @@ When multiple catalogs provide the same package, you can set priorities to resol
 In your `ClusterCatalog` resource, set the `priority` field:
 
 ```yaml
-apiVersion: catalogd.operatorframework.io/v1alpha1
+apiVersion: clustercatalogs.olm.operatorframework.io
```

Comment:
Drop the `clustercatalogs.` prefix. That's the plural resource name, not part of the apiGroup.

Reply:
Oh boy, you have a good catcher too :

I did the test right but I did the replace wrong :-P 

```
) $ kubectl get crds | grep clustercatalog
clustercatalogs.olm.operatorframework.io     2024-11-06T21:34:38Z
camiladeomacedo@camilas-MacBook-Air ~/go/src/github.com/operator-framework/operator-controller (fix-install-script) $ kubectl apply -f - <<EOF
> apiVersion: olm.operatorframework.io/v1alpha1
> kind: ClusterCatalog
> metadata:
>   name: operatorhubio
> spec:
>   source:
>     type: Image
>     image:
>       ref: quay.io/operatorhubio/catalog:latest
>       pollInterval: 10m
> EOF
clustercatalog.olm.operatorframework.io/operatorhubio configured
```

Reply:
Done! Thanks.
Stopping for today to avoid missing any boats—at least for today. :-)